build/build.go

```diff
@@ -627,7 +627,7 @@ func toSolveOpt(ctx context.Context, node builder.Node, multiDriver bool, opt Op
 			pp[i] = platforms.Format(p)
 		}
 		if len(pp) > 1 && !nodeDriver.Features(ctx)[driver.MultiPlatform] {
-			return nil, nil, notSupported(nodeDriver, driver.MultiPlatform)
+			return nil, nil, notSupported(driver.MultiPlatform, nodeDriver, "https://docs.docker.com/go/build-multi-platform")
```

Comment:
Do the go-redirect URLs need a trailing slash?

Reply:
Not strictly needed; the redirect handles both cases: https://github.com/docker/docs/commit/bf4a605ad14a79b9e4e935ded0e0b0773adc739f

But probably a good idea to add them. Let me fix that.

Reply:
Ah, yes, I was aware we were _able_ to handle both, but I think we use URLs with a trailing `/` as the "canonical" one for all our links, so thought I'd be better to be consistent and use URLs ending with a `/`. (For the non-`/go/` URLs this prevents an extra redirect).